packages/kbn-search-connectors/types/connectors.ts

```diff
@@ -68,7 +68,9 @@ export type ConnectorConfiguration = Record<
 > & {
   extract_full_html?: { label: string; value: boolean }; // This only exists for Crawler
   use_document_level_security?: ConnectorConfigProperties;
-  use_text_extraction_service?: ConnectorConfigProperties; // This only exists for SharePoint Online
+  use_text_extraction_service?: ConnectorConfigProperties;
+  enumerate_all_sites?: ConnectorConfigProperties; // This only exist for Sharepoint Online
```

Comment:
These (enumerate_all_sites and fetch_subsites) should just be part of the SharePoint online config, not the shared configuration definition.

Reply:
Should I create a new type in that case ?

Reply:
I mean it shouldn't need a separate type, the only reason we have `extract_full_html` and `use_document_level_security` there is becuase we have some specific logic that relies on those fields existing, but config fields like this should just fit the regular `ConnectorConfiguration` type that already exists. You shouldn't need to explicitly add these fields to any type.

Reply:
yes, you are right. Looks like my toolchain stops working in every node release, it reported errors which doesn't exist anymore 🤷